core/server/master/src/main/java/alluxio/master/metastore/rocks/RocksInodeStore.java

```diff
@@ -442,6 +471,9 @@ public Long next() {
 
   @Override
   public Optional<Inode> getChild(Long inodeId, String name, ReadOption option) {
+    if (mClosed.get()) {
+      throw new RuntimeException("RocksDB is closed. Master is failing over or shutting down.");
```

Comment:
Honestly I'm not sure this is the best exception to throw. Feel free to comment if you got a better one :)

Reply:
Maybe alluxio.exception.runtime.UnavailableRuntimeException